src/UiPath.Workflow.Runtime/Runtime/CallbackWrapper.cs

```diff
@@ -56,13 +55,13 @@ internal string SerializedDeclaringTypeName
     }
 
     [DataMember(Name = "ActivityInstance")]
-    internal ActivityInstance SerializedActivityInstance
+    public ActivityInstance SerializedActivityInstance
```

Comment:
Is this needed as public?

Please go through the PRs again, making sure every change can be justified.

Reply:
Reverted SerializedActivityInstance to be internal, because isn't used anywhere in our repo.